dashboard/pages/correlation/stages/s2_correlation_plots.py

```diff
@@ -93,6 +93,7 @@
                                 ),
                             ],
                         ),
+                        html.Hr(),
```

Comment:
You can remove it and change the div below to:
```
html.Div(
    className="mt-3 mb-1 d-flex justify-content-center",
    children=[
        html.Button(
            make_download_button_text(
                "Save filtered dataframe "
            ),
            className="btn btn-primary btn-lg btn-block btn-report",
            id="save-filtered-button",
        ),
        dcc.Download(id="download-filtered-csv"),
    ],
),
```
![obraz](https://github.com/zuzg/drug-screening/assets/82370491/643dd4c9-13b6-45ca-b4d8-9359ce718a17)